ansible/fatimage.yml

```diff
@@ -101,12 +101,6 @@
         tasks_from: install.yml
       when: "'filebeat' in group_names"
 
```

Comment:
We should pull the containers for kube-prom-stack - maybe add an install.yml in the role which does that.

E.g. see https://github.com/stackhpc/ansible-slurm-appliance/blob/main/ansible/roles/mysql/tasks/install.yml#L18 BUT
note there is a k3s-specific way of using pre-pulled images.

Reply:
the issue is that k3s uses an embedded containerd runtime, meaning we can'r pre-pull images without starting k3s, which we don't want to do on build. There is a way to use an external containerd runtime but I can't seem to get it to work. We could potentially pre-pull images with podman and export them as archives, then import into k3s in site.yml, but we probably want to get rid off podman eventually. At the moment all of the image sizes in kube-prometheus-stack sum to less than 1Gb so this might not be worth the effort